en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
 topics: [r]
 difficulty: 3
 activity: analyzing
-abstract: In this lesson, you will learn how to download YouTube comments and use the R programming language to analyze the political valences of a corpus with the Wordfish algorithm.
+abstract: In this lesson, you will learn how to download YouTube comments and use the R programming language to analyze the political values of a corpus with the Wordfish algorithm.

```

Comment:
I had not come across the term 'political valences' before reading this lesson, and I wasn't sure if it was a typo or a new word. I actually struggled to find a definition online – it doesn't seem to be a common term. 

Not all readers will be familiar with this concept either, so I suggest removing it from the abstract, and only introducing it later in the lesson accompanied by an explanatory sentence.

Reply:
"political values of a corpus" doesn't quite capture what we mean. how about "the opposing ideological underpinnings within a corpus"?